### DISCUSSION TITLE
Problem when running the simulation

### DISCUSSION THREAD

**Discussion by alfassyn** [surgical-robotics-ai/surgical_robotics_challenge#83] [Q&A]
Hi, I also have a problem when run the simulation. I use this ./ambf_simulator --launch_file ~/surgical_robotics_challenge>/launch.yaml -l 0,1,3,4,14,15 -p 120 -t 1 --override_max_comm_freq 120 and the simulation only shows white windows. I use Ubuntu 20.04 and ROS Noetic. Is there any way to fix the problem? Thank you
<img width="960" alt="Screenshot 2023-06-16 133556" src="https://github.com/collaborative-robotics/surgical_robotics_challenge/assets/136787707/784a28bd-65ff-4883-a043-833329e40107">

**Comment by adnanmunawar** [comment]
Hello, please read the instructions again. The `<surgical_robotics_challenge>` part needs to be replaced with the location of where you cloned this repo.
